data/core/units/elves/Hero.cfg

```diff
@@ -15,7 +15,7 @@
     advances_to=Elvish Champion
     cost=30
     usage=fighter
-    description= _ "Elves, though not warlike by nature, are swift to learn; a relatively small amount of experience will turn an elf from a competent fighter into a master of combat. Those who are honored as heroes are strong with both sword and bow, and their skill once acquired does not fade for lack of practice."
+    description= _ "Elves that have grown more adept at swordsmanship often forego a shield in favor of a secondary shortsword, useful for both parrying enemy attacks and sneaking in extra strikes when engaging in melee combat. These dimuitive blades are often known as 'Västaja', little heroes, for their third use as replacement swords for weaponless allies. More inexperienced fighters are easily disarmed by stronger opponents and quickly overwhelmed when only left with a bow to fight with. By offering their parrying swords to their more vulnerable comrades, elvish heroes grant them a second chance to fight or to retreat with their lives."
```

Comment:
> These dimuitive blades

Did you mean diminute? isn't that a bit weird for a sword to be called diminute? it makes it sound like it's a toothpick. Maybe you could just say "These shorter blades".

Reply:
That would be diminutive, I must have made a typo. Diminutive is often used as a word to indicate affection and can be less strong compared to "tiny" (i.e. something diminutive would be larger than something tiny, depends on context really). I'm using it here to indicate that despite these swords being small, they are still very useful and precious.